src/scss/mixins/_labels.scss

```diff
@@ -1,10 +1,9 @@
 /// Label styles
 @mixin oFormsLabel {
-	@include oTypographySansBold(1);
+	@include oTypographyBold('sans');
```

Comment:
Just curious, why the change?

Reply:
@gvonkoss 
Alice did this as:
>The rest of the font props are being set elsewhere so this only needs
to set the correct weight which can be done with this mixin.
https://github.com/Financial-Times/o-forms/pull/170/commits/7326e0a79f3b91b716088b27f3f9efdc71540685

I see `oFormsCommonFieldBase` sets the font-family. I think it should be using `oTypographySans` but don't think we can switch to that without it being breaking as `$o-forms-font-family` is a public variable. If you agree that's how it should be, we could create an issue set the font family with `oTypographySans`?